doc/self-hosting.md

```diff
@@ -184,9 +184,14 @@ Start the server using the Sentry DSN. You can set it:
 sudo SENTRY_DSN=https://xxx:yyy@sentry.io/zzz node server
 ```
 
-- or by `sentry_dsn` secret property defined in `private/secret.json`
+Or via [checked in config](server-secrets.md):
```

Comment:
Bit verbose, but maybe something like
```suggestion
Or via [any of the other supported options](server-secrets.md):
```

Reply:
This feels even more confusing to me.

Reply:
It makes sense to me in the broader context, with the preceding talking about setting the environment variable and then the `or..` but I like your suggested text better any way @PyvesB